examples/widgets/resolution_awareness.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) 2016, the Flutter project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+import 'dart:ui' as ui;
+
+import 'package:flutter/services.dart';
+import 'package:flutter/widgets.dart';
+
+class ExampleApp extends StatefulComponent {
+  ExampleState createState() => new ExampleState();
+}
+
+List<double> ratios = <double>[ 1.0, 1.8, 1.3, 2.4, 2.5, 2.6, 3.9 ];
```

Comment:
You just need to const the literal as well.

`const List<double> ratios = const <double>[ ... ]`